src/Caching/MemcachedServiceProvider.php

```diff
@@ -70,12 +70,11 @@ public function register(Container $app)
                     break;
             }
 
-            $logger = $this->getLogger($app, $name);
-            if ($logger) {
+            if (null !== $logger = $this->getLogger($app, $name)) {
```

Comment:
It's your project, so sure. But I'm not a fan of this ;).
Sure it works but it's harder to scan these kinds of lines. 

But than again, maybe it's okay because it's not a project but a library..